doc/source/user_guide/boundary_conditions.rst

```diff
@@ -53,25 +59,26 @@ The following example demonstrates how you can define boundary conditions using
         'quit'
     )
 
-Copying Boundary Conditions
+Copying boundary conditions
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
-define/boundary-conditions/copy-bc TUI: Copies boundary conditions to other zones.
```

Comment:
@ypatel-qa Hey, did you mean to document these with the slashes and dash characters as in the Fluent TUI console?  If so, then what we might do is write this a bit different....
```
This Fluent TUI console command:

.. code:: scheme

   define/boundary-conditions/list-zones

prints in the console window the types and IDs of all zones.  The equivalent Python command is

.. code:: python

  session.solver.tui.define.boundary_conditions.list_zones()
```

Reply:
That's fine. Just know that the Google styleguide check is not going to accept splitting the sentence with a code block. Plus, it really makes it  much harder to read. For your proposal above, I'd suggest rewriting it as:

Here are the Fluent TUI console command and the Python code for printing in the Fluent console the types and IDs of all zones.

**TUI console command**

.. code:: scheme

   define/boundary-conditions/list-zones

**Python code**

.. code:: python

  session.solver.tui.define.boundary_conditions.list_zones()

Reply:
@dnwillia-work @PipKat That is correct. I meant to refer to the equivalent TUI command.